README.md

```diff
@@ -27,35 +31,47 @@ cd ../../../
 rm -rf cpprestsdk
 ```
 
-### Windows
-Windows dependencies are managed through Vcpkg and Nuget.
-
-```
-vcpkg install cpprestsdk:x64-windows
-vcpkg install zlib:x64-windows
-vcpkg install boost-system:x64-windows
-vcpkg install boost-program-options:x64-windows
-vcpkg install boost-test:x64-windows
-vcpkg install boost-uuid:x64-windows
-```
-
-## Build
+### Build
 
 ```
 mkdir build
 cd build
 cmake ..
 make
+```
 
-# Test
+### Test
+```
 mkdir build
 cd build
 cmake .. -DTURN_OFF_DEVIRTUALIZE=On
 make rl_clientlib_test
 make test
 ```
 
+## Windows
+Windows dependencies are managed through Vcpkg and Nuget.
+
+```
+vcpkg install cpprestsdk:x64-windows
+```
+
+### Build + Test
+
+Open `rl.sln` in Visual Studio 2017.
+
 ### Experimental - CMake on Windows
```

Comment:
>Experimental [](start = 4, length = 12)

Experimental .. as in is this an alternate optional way to build the project? If that is the case, could we call this out in a separate section so it is not confused with the rest of how configure/build works on Windows?

Reply:
Or maybe it should say ## Experimental instead of ### Experimental on line 63. I am trying to understand if this section is required or some optional way of building on Windows. If it is optional, also call that out explicitly.

---
In reply to: [237568116](https://github.com/VowpalWabbit/reinforcement_learning/pull/25#discussion_r237568116) [](ancestors = 237568116)

Reply:
I think it should stay at the same nesting level as there is a section for Windows and Ubuntu. I've added a comment.